### PR TITLE
Select all columns like other validation errors

### PR DESCRIPTION
**Pull request by Levia**
## Description

* Fix issue where a validation error doesn't show realted records due to missing columns in query SELECT statement

## Notes

[Related Codebase ticket](https://unep-wcmc.codebasehq.com/projects/species-rails-4-upgrade/tickets/39)

### CODE REVIEW

**Comment by Levia** [comment]
@lucacug This is ready to be quickly looked at if you have some time please. Thanks!

File: app/models/trade/taxon_concept_source_validation_rule.rb [hunk 1]
```diff
@@ -47,10 +47,7 @@ def matching_records_arel(table_name)
     )
 
     arel = s.project(
-      s['id'],
-      s['taxon_concept_id'],
-      s['source_code'],
-      s['accepted_taxon_name']
+      s['*']
```
Comment (lucacug) [inline app/models/trade/taxon_concept_source_validation_rule.rb:50]: I'm not very familiar with this syntax nor with the Arel class, but I guess this is like doing `SELECT * FROM`. If so happy for this to be merged.

**Comment by lucacug** [comment]
@Levia  left a small comment, feel free to merge this yourself if the comment makes sense

File: app/models/trade/taxon_concept_source_validation_rule.rb (hunk 1 above)
Comment (Levia) [reply app/models/trade/taxon_concept_source_validation_rule.rb:50]: Yep that's exactly it. I'll merge it then :) Thanks!

**Pull request merged**